api/woeip/apps/air_quality/tests/test_api.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from rest_framework.test import APIRequestFactory
+
+from woeip.apps.air_quality import views
+from woeip.apps.air_quality.tests import factories
+
+request_factory = APIRequestFactory()
+
+
+@pytest.mark.django_db
+def test_collection_list():
+    collection_file = factories.CollectionFileFactory()
```

Comment:
Lines 12-22 seem to be unrelated to the collection list test, and nothing happens with `stored_values` after the loop - is this code for another test?

Reply:
Yeah, this is a bit unfinished. I'd like to add another test that makes sure that the points stored to the db are the same that get returned by the API call. Will work on that now!

Reply:
Well the hardest part was finding where this was documented, but looks good now.

Reply:
Looks great! I think it might be good to break this up into multiple tests, so that if a test 
fails it's immediately obvious what the problem is. 
Perhaps a good way to organize might be a `TestCollection` class, e.g.

```
class TestCollectionViews(APITestCase):
    @pytest.mark.django_db
    def setUp(self):
        self.collection_file = factories.CollectionFileFactory()
        self.pollutant = factories.PollutantFactory()
        self.n_values = 10
        self.test_values = []
        for _ in range(self.n_values):
            pollutant_value = factories.PollutantValueFactory(
                collection_file=self.collection_file,
                pollutant=self.pollutant,
            )
            self.test_values.append(pollutant_value.value)

    @pytest.mark.django_db
    def test_get_collection_list(self):
        request = request_factory.get("/collection")
        view = views.CollectionViewSet.as_view({"get": "list"})
        response = view(request)
        assert response.status_code == 200

    @pytest.mark.django_db
    def test_get_collection_retrieve(self):
        request = request_factory.get("/collection")
        view = views.CollectionViewSet.as_view(actions={"get": "retrieve"})
        response = view(request, pk=self.collection_file.collection.pk)
        assert response.status_code == 200

    @pytest.mark.django_db
    def test_get_collection_data(self):
        request = request_factory.get("/collection")
        view = views.CollectionViewSet.as_view(actions={"get": "data"})
        response = view(request, pk=self.collection_file.collection.pk)
        assert response.status_code == 200
        stored_values = [pv["value"] for pv in response.data["pollutant_values"]]
        assert stored_values == self.test_values
```

Reply:
Also, going to approve the PR since this is a non-blocking improvement suggestion.